src/components/projects/ProjectCard.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import '../../css/ProjectsPage.css';  // You will design styles here
+
+export default function ProjectCard({ project }) {
+  return (
+    <div className="project-card">
+      <a href={project.url} target="_blank" rel="noopener noreferrer">
+        {/* Uncomment the below once you have images */}
+        {/* <div className="project-card-image">
```

Comment:
let's try floating this left of the `h3`, or using a grid if that's cleaner. I'd like to get these added ASAP. We have some good ones already, and it would be great to have a default image of some sort. I know we talked about this, and you had some ideas. I'm open to options, but I'd rather have something. I'm focused on this because we need to be able to quickly add small projects, and adapt to working with external groups as needed. I don't want this to ever become a roadblock to getting a project started and making it look good.